cmd/argocd/commands/app.go

```diff
@@ -348,6 +363,18 @@ func NewApplicationLogsCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	return command
 }
 
+// This is used in failure retry - calculate new sinceSeconds based on lastTimeSteampReceived
+func getNewSinceSeconds(lastTimeStampReceived string, sinceScondsDefault int64) int64 {
```

Comment:
Just a nitpick: It should be `sinceSecondsDefault` instead of `sinceScondsDefault`

Reply:
Corrected it. Thank you Jann!